skimage/image_quality_measure.py

```diff
@@ -0,0 +1,53 @@
+from skimage.util import img_as_float
+import numpy as np
+from scipy.ndimage import maximum_filter, minimum_filter
+
+
+def quality(image: np.ndarray,
+            size: int = 3) -> float:
+    """
+        The image quality measure called EME based on [1]_.
+
+        Parameters
+        ----------
+        image : array
+            Input image of which the quality should be assessed.
+            Can be either 3-channel RGB or 1-channel grayscale.
+            The function converts pixel intensities into floats by default.
+        size : int
+            Size of the window. Default value is 3.
+
+        Returns
+        -------
+        eme : float
+            The number describing image quality.
+
+        References
+        ----------
+        .. [1] Agaian, Sos S., Karen Panetta, and Artyom M. Grigoryan.
+               "A new measure of image enhancement."
+               IASTED International Conference on Signal Processing
+               & Communication. Citeseer, 2000.
+        Examples
+        --------
+        >>> from skimage.data import camera
+        >>> from skimage.exposure import equalize_hist
+        >>> img = camera()
+        >>> print("Image quality:\n")
+            Image quality:
+        >>> print(f"\tbefore: {quality(img)}")
+                before: 0.9096745071475523
+        >>> print(f"\tafter: {quality(equalize_hist(img))}")
+                after: 1.299327371881219
```

Comment:
The docstring formatting is a little unconventional which may be the reason for the doctest failure on travis. What is you reasoning for using print functions? I think the indentation after the print statements confuses the parser. You could use comments instead to annotate "before" and "after". E.g.:

```python
>>> quality(img)  # before
0.9096745071475523
>>> quality(equalize_hist(img))  # after
1.299327371881219
```

And just a nitpick: could you remove one indentation level from the docstring? It's not really necessary.

Reply:
Oh, I commented in the wrong file. But it applies to the other one as well.

Reply:
Thanks for the comments @lagru, I did not really think about why I put the print statements there. Makes sense to remove them in this contexts, true.